.github/workflows/gradle.yml

```diff
@@ -292,7 +292,7 @@ jobs:
             ./gradlew :vividus-tests:runStories -x testVividusInitialization \
                                                 -Pvividus.configuration.environments=system/mobile_app/browserstack/ios \
                                                 -Pvividus.configuration.suite=system/mobile_app/healthcheck \
-                                                -Pvividus.configuration.profiles=mobile_app/ios,browserstack/mobile_app \
```

Comment:
Noisy changes?

Reply:
browserstack/mobile_app should go first to override data in mobile_app/iOS, as we did this in saucelabs